nrpy/infrastructures/CarpetX/general_relativity/enforce_detgammahat_constraint.py

```diff
@@ -0,0 +1,131 @@
+"""
+Generates function to enforce the det(gammabar) = det(gammahat) constraint..
+
+Authors: Zachariah B. Etienne
+        zachetie **at** gmail **dot* com
+         Samuel Cupp
+"""
+
+from typing import Union, cast, List
+from inspect import currentframe as cfr
+from types import FrameType as FT
+import sympy
```

Comment:
please use `import sympy as sp`, for consistency across codebase.

Reply:
changed